provider/pkg/gen/dotnet-templates/Kustomize/Directory.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2016-2020, Pulumi Corporation
```

Comment:
It doesn't appear this has an auto-generated warning like the other languages do. Or was this manually written?

Reply:
Will add